src/components/InventoryGroups/Modals/DeleteGroupModal.cy.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { mount } from '@cypress/react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import { getStore } from '../../../store';
+import DeleteGroupModal from './DeleteGroupModal';
+
+describe('Delete Group Modal', () => {
+    before(() => {
+        cy.window().then(window => window.insights = {
+            chrome: {
+                isProd: false,
+                auth: {
+                    getUser: () => {
+                        return Promise.resolve({});
+                    }
+                }
+            }
+        });
+    });
+
+    beforeEach(() => {
+        cy.intercept('DELETE', '**/api/inventory/v1/groups/1', {
+            statusCode: 200, body: {
+            }
+        }).as('delete');
+
+        mount(
+            <MemoryRouter>
+                <Provider store={getStore()}>
+                    <DeleteGroupModal isModalOpen={true}/>
```

Comment:
```suggestion
                    <DeleteGroupModal isModalOpen={true} modalState={ { id: 1, name: 'name' } }/>
```
^ I find it better rather than defining the default state in the component itself and making your tests dependent on the component default data, not on the data you can provide to it.

Reply:
Agree this is much better